.styleci.yml

```diff
@@ -3,13 +3,4 @@ preset: recommended
 linting: true
 
 finder:
-    exclude:
-        - "tests"
-    name:
-        - "*.php"
-    path:
-        - "src"
-    not-path:
-        - "tests"
-    depth:
-        - "< 3"
```

Comment:
https://github.com/StyleCI/Config/blob/master/src/ConfigFactory.php#L74 let's you see the config defaults.